js/modules/loki_rpc.js

```diff
@@ -494,11 +521,41 @@ const lokiFetch = async (url, options = {}, targetNode = null) => {
         );
 
         if (result === BAD_PATH) {
-          log.error('[path] Error on the path');
+          const pathArr = [];
+          path.forEach(node => {
+            pathArr.push(`${node.ip}:${node.port}`);
+          });
+          log.error(
+            `[path] Error on the path: ${pathArr.join(', ')} to ${
+              targetNode.ip
+            }:${targetNode.port}`
+          );
           lokiSnodeAPI.markPathAsBad(path);
+          return false;
+        }
+
+        // result maybe false
+        if (result) {
```

Comment:
We can use [`result.ok`](https://developer.mozilla.org/en-US/docs/Web/API/Response/ok) to determine if the status code is within the success range (200 - 299)

Reply:
the proxy/OR don't include the `ok` property when we fake a response object

Reply:
Ah ok fair enough 👍